app/controllers/application_controller.rb

```diff
@@ -41,6 +41,10 @@ def routing_error
         Common::Exceptions::ClientError.new(exception.message, additional_attributes)
       when Breakers::OutageException
         Common::Exceptions::ServiceOutage.new(exception.outage)
+      when Common::Client::Errors::Error
+        # SSLError, ConnectionFailed, SerializationError, etc
+        Raven.capture_exception(exception.cause) if ENV['SENTRY_DSN'].present? && exception.cause.present?
```

Comment:
Yes. It would. But I couldn't think of a better way to make sure it is the original cause of the exception. The backtrace kind of gets convoluted with rescue + reraise.